src/components/Layouts/Layouts.stories.js

```diff
@@ -0,0 +1,269 @@
+/**
+ * @file Modular layout stories.
+ * @copyright IBM Security 2020
+ */
+
+import { boolean } from '@storybook/addon-knobs';
+import { storiesOf } from '@storybook/react';
+
+import React from 'react';
+import { Grid, Row, Column } from 'carbon-components-react';
+import ArrowRight20 from '@carbon/icons-react/lib/arrow--right/20';
+
+import {
+  Accordion,
+  AccordionItem,
+  Button,
+  Tab,
+  Tabs,
+  TypeLayout,
+  TypeLayoutBody,
+  TypeLayoutRow,
+  TypeLayoutCell,
+} from '../..';
+
+import { disableCentered, patterns } from '../../../.storybook';
+
+const toggleDebugging = () => boolean('Toggle debugging 🕵️‍♀️', false);
+
+const showAccordion = () => boolean('Show accordion', true);
+const showButtonGroup = () => boolean('Show button group', true);
+const showCardGroup = () => boolean('Show card group', true);
+const showTabGroup = () => boolean('Show tab group', true);
+
+const showFirstRow = showCardGroup() || showButtonGroup();
+const showSecondRow = showTabGroup() || showAccordion();
+
+disableCentered(storiesOf(patterns('Modular Layouts'), module)).add(
+  'details page layout',
+  () => (
+    <main className={toggleDebugging() && 'security--debug'}>
+      <h1 className="bx--type-productive-heading-04">productive-heading-04</h1>
+      <Grid condensed>
+        {showFirstRow && (
+          <Row>
+            {showCardGroup() && (
+              <Column md={3} lg={6}>
+                <h2
+                  style={{ margin: 0 }}
```

Comment:
@SimonFinney related to your review comment: https://github.com/carbon-design-system/ibm-security/pull/504#pullrequestreview-402513118

Not sure the best say to go about resetting these `h2` margin defaults... 

Should we start exporting some heading reset styles (just resetting margins and padding, before applying specific padding via these generated classes)? Seems like the direction to go... if the end goal is to have folks apply these layouts + classes as-is, then we've want to ensure that the headings are styled correctly. 🤔

Reply:
There are pros and cons to each approach - reset utility classes would provide additional control but an extra cognitive step for consumers and potentially more room for user error.

An alternative could be to use margins instead of padding for the layout classes, but I don't fully understand what the implications of that would have on the larger system you're proposing.

I suspect @cameroncalder could provide insight and work through some of these details that are arising as we understand it better.